railties/lib/rails/application/configuration.rb

```diff
@@ -312,6 +312,11 @@ def load_defaults(target_version)
           if respond_to?(:action_controller)
             action_controller.allow_deprecated_parameters_hash_equality = false
           end
+
+          if respond_to?(:action_view)
+            action_view.form_with_generates_remote_forms = false
+            action_view.form_with_generates_local_forms = false
```

Comment:
I feel like this should be true? Because the old setting defaults to false, and I assume you want the default behaviours to be the same from the old setting to the new one.